### PR TITLE
add new images to array

### PR DESCRIPTION
**Pull request by edwardkrupicka**
A bunch of new images

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -194,6 +194,116 @@ app.locals.images = [
...
+    title: 'STOP SEXUALIZING GIRLS',
+    color: ['black', 'grey', 'white'],
+    artist: 'K. Mitch Hodge',
+    type: 'photography',
+    favorited: false,
+    quantity: 0,
+    price: '25'
+  },
+  {
+    id: "15",
```
Comment (tanya2108) [inline server.js:297]: I like this one

**Review by tanya2108 — APPROVED** [review]
Thanks for adding more images

**Review by delilahrois — APPROVED** [review]
Looks great, now we'll have way more art to work with!

**Review by reganlosey — APPROVED** [review]
Looks good

**Pull request merged**